bin/fury

```diff
@@ -7,4 +7,8 @@ require 'rubygems'
 require 'gemfury'
 require 'gemfury/command'
 
+if defined?(Warning) && Warning.respond_to?('[]'.to_sym)
```

Comment:
@rykov had to add additional qualifier, since it looks like Warning categories is very recent add, only in 2.7.0.p0 / 2.7.1.

Reply:
👍, but can you leave `respond_to?('[]')` just as string? Save an allocation and a couple CPU cycles? 🤣

Reply:
Copy copy. Updated.